views/index.html

```diff
@@ -11,7 +11,7 @@ <h2 id="message-title">Thank you</h2>
 </span>
 
 <section id="join" class="u-full-width header">
-    <h1><b>Join</b> dvcoders</h1>
+    <h1><b>Join</b> Dvcoders</h1>
```

Comment:
I agree. I should read the constitution.

On Tue, Sep 13, 2016 at 4:18 PM, Jake Loo <notifications@github.com> wrote:

> In views/index.html
> <https://github.com/dvcoders/dvcoders-backend/pull/43#discussion_r78662508>
> :
>
> > @@ -11,7 +11,7 @@ <h2 id="message-title">Thank you</h2>
> >  </span>
> >
> >  <section id="join" class="u-full-width header">
> > -    <h1><b>Join</b> dvcoders</h1>
> > +    <h1><b>Join</b> Dvcoders</h1>
>
> https://github.com/dvcoders/constitution/blob/master/
> constitution.md#name-of-organization
>
> The name of this organization shall be dvcoders
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dvcoders/dvcoders-backend/pull/43/files/4fdba9deca48cd157b27d367aa12b78d75a03d64#r78662508>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQrQh552tO8drFn9iG1f8pIeli2MYHhbks5qpy9QgaJpZM4J8N8n>
> .
>